src/metrics/management/commands/fetch_forward_links.py

```diff
@@ -0,0 +1,143 @@
+import requests
+from bs4 import BeautifulSoup
+
+from django.core.management.base import BaseCommand
+
+from utils import setting_handler
+from journal import models as jm
+from submission import models as sm
+from metrics import models
+
+
+def process_article(link, article):
+    print('Processing article', end='...')
+    doi = link.doi.contents[0]
+    volume = link.volume.contents[0] if link.volume else ''
+    issue = link.issue.contents[0] if link.issue else ''
+    issn = link.issn.contents[0] if link.issn else None
+
+    defaults = {
+        'year': link.year.contents[0],
+        'journal_title': link.journal_title.contents[0],
+        'article_title': link.article_title.contents[0],
+        'volume': volume,
+        'issue': issue,
+        'journal_issn': issn,
+    }
+
+    models.ArticleLink.objects.get_or_create(
+        article=article,
+        doi=doi,
+        object_type='article',
+        defaults=defaults,
+    )
```

Comment:
Might be worth also updating the object if any of the metadata attributes have changed

Reply:
Yes, added that to a new issue as want to get this thru: https://github.com/BirkbeckCTP/janeway/issues/1353